torchmultimodal/models/mdetr/model.py

```diff
@@ -257,3 +260,236 @@ def mdetr_resnet101(
         extra_query_embeddings,
     )
     return mdetr
+
+
+class MultiHead(nn.Module):
```

Comment:
what was the reason for moving these out of examples?

Reply:
Trying to follow the convention with FLAVA of having task-specific model classes in core (as long as it makes sense). For MultiHead specifically, it is just a ModuleDict wrapper and gets removed in the next PR